src/collections/Menu/index.d.ts

```diff
@@ -135,7 +135,7 @@ interface MenuItemProps extends ReactMouseEvents<HTMLElement> {
   color?: SemanticCOLORS;
 
   /** Shorthand for primary content. */
-  content?: string;
+  content?: any;
```

Comment:
Per @layershifter in https://github.com/Semantic-Org/Semantic-UI-React/pull/1159#issuecomment-272907848, let's use `React.Node`.